sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
I came to a conclusion that Android Studio does not "see" classes outside _library/src/main/android_ directory for some reason. I mean, when I move interfaces from _library/src/main/java_ directory to _library/src/main/android_, they become visible. I'm still searching for an explanation of this behavior. Any ideas?